frontend/src/features/projects/erp/forms/EnhancedReferralCompleteForm.tsx

```diff
@@ -50,6 +51,9 @@ const EnhancedReferralCompleteForm = ({
   const [proceedToSpl, setProceedToSpl] = useState(false);
   const [notInSpl, setNotInSpl] = useState(false);
   const [disposeExternally, setDisposeExternally] = useState(false);
+  const isClearanceNotificationSentOnRequired = clearanceNotificationSentOnRequired(
```

Comment:
since this is only ever used in the below checks for disabled, would probably make more sense to include the !formikProps.values.clearanceNotificationSentOn in this boolean, so you can reduce the amount of duplicated code in the disabled checks.